### PR TITLE
Don't add not accessable courses to the array.

### PR DESCRIPTION
**Pull request by ikorgik**

### CODE REVIEW

File: src/Controller/CourseListController.php [hunk 1]
```diff
@@ -61,7 +61,10 @@ public function build() {
 
     $normalizedCourses = [];
     foreach ($courses as $course) {
-      $normalizedCourses[] = $this->normalizeNode($course);
+      $normalizedCourse = $this->normalizeNode($course);
```
Comment (spleshka) [inline src/Controller/CourseListController.php]: How come they are were loaded but not accessible? Should be resolved before normalizing it IMO

Comment (spleshka) [reply src/Controller/CourseListController.php]: @ikorgik

Comment (ikorgik) [reply src/Controller/CourseListController.php]: Added @todo to include access check when we will be configuring global permissions. Don't want to spend time on it now

Comment (ikorgik) [reply src/Controller/CourseListController.php]: @spleshka

**Pull request merged**